scripts/haas

```diff
@@ -14,5 +14,8 @@
 # express or implied.  See the License for the specific language
 # governing permissions and limitations under the License.
 
+import sys
+
 from haas import cli
-cli.main()
+
+sys.exit(cli.main())
```

Comment:
cli.main() doesn't return a value currently.

I was thinking that maybe cli.main() should do try/except with the proper exceptions, then call sys.exit() itself.

What do you think?